### PR TITLE
created store config file, removed unused constants in header and footer

### PR DESCRIPTION
**Pull request by mitdralla**

### CODE REVIEW

**Comment by mitdralla** [comment]
This PR cleans up all current eslint errors. There is a new object added to config.jsx which we can use to define store settings and config. This can be used to toggle on or off features, numbers of things to display, links and their orders, etc.

**Comment by mitdralla** [comment]
This also fixes the Travis build check to a passing status.

File: src/config.jsx [hunk 1]
```diff
@@ -1 +1,145 @@
-const HERO_CAROUSEL_MAX_ITEMS = 10
+// dApp Store Configuration File.
```
Comment (gregnorth) [inline src/config.jsx:1]: 👍

**Review by gregnorth — APPROVED** [review]
LGTM

**Pull request merged**